packages/waku/src/lib/middleware/rsc/utils.node.ts

```diff
@@ -0,0 +1,3 @@
+import fsPromises from 'node:fs/promises';
+export const fileExists = async (path: string) =>
+  !!(await fsPromises.stat(path).catch(() => false));
```

Comment:
Let's use sync version for exists.

I need to restructure the code base, but I'd like to avoid "utils".